crates/sui-indexer-alt/src/args.rs

```diff
@@ -68,12 +66,4 @@ pub struct ConsistencyConfig {
 impl ConsistencyConfig {
     const DEFAULT_CONSISTENT_PRUNING_INTERVAL_MS: u64 = 300_000;
     const DEFAULT_PRUNER_DELAY_MS: u64 = 120_000;
-
-    pub fn consistent_pruning_interval(&self) -> Duration {
-        Duration::from_millis(self.consistent_pruning_interval_ms)
-    }
-
-    pub fn pruner_delay(&self) -> Duration {
-        Duration::from_millis(self.pruner_delay_ms)
-    }
```

Comment:
Why does this commit remove what was added in the previous one? unintentional?

Reply:
Oh...maybe i see that this was intentional, its slightly confusing to have the default exist in the ConsistencyConfig but maybe this will make sense at the end of your whole stack